drivers/bridge/labels.go

```diff
@@ -15,4 +15,7 @@ const (
 
 	// DefaultBridge label
 	DefaultBridge = "com.docker.network.bridge.default_bridge"
+
+	// ContainerIfacePrefix label
+	ContainerIfacePrefix = "com.docker.network.bridge.container_iface_prefix"
```

Comment:
Looking at this again, it makes me think people may want the same option while using different network drivers in future.

In a way it s not specific to a network driver (like the MTU option we have now).

So I was thinking it may be better we make this option driver agnostic from the beginning, and define it in https://github.com/docker/libnetwork/blob/master/netlabel/labels.go

as 

`ContainerIfacePrefix = "com.docker.network.container_iface_prefix`

Also wondering whether we should keep a 1-to-1 match to the networking objects (here would be `sandbox_iface_PREFIX`) or stick to `container` given it is a user input option. It is probably better to leave it as `container_iface_prefix`, more clear.